### PR TITLE
NumberSet Class

### PR DESCRIPTION
**Pull request by tharmaman**

### CODE REVIEW

File: src/foam/core/NumberSet.js [hunk 1]
```diff
@@ -0,0 +1,107 @@
+foam.CLASS({
+  package: 'foam.core',
+  name: 'NumberSet',
+
+  documentation: 'An array of String values.',
```
Comment (kgrgreer) [inline src/foam/core/NumberSet.js]: Looks like a cut&paste error.

File: src/foam/core/NumberSet.js [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+  label: 'List of text strings',
```
Comment (kgrgreer) [inline src/foam/core/NumberSet.js]: same

File: tools/classes.js [hunk 3]
```diff
@@ -623,7 +623,10 @@ var classes = [
   'foam.nanos.approval.FulfilledApprovablePredicate',
   'foam.nanos.approval.FulfilledApprovableRule',
   'foam.nanos.approval.AuthenticatedApprovalDAOAuthorizer',
-  'foam.nanos.approval.PopulateApprovalRequestsDAO'
+  'foam.nanos.approval.PopulateApprovalRequestsDAO',
+
+  // holders
```
Comment (kgrgreer) [inline tools/classes.js]: It isn't really a holder no. More like StringArray. Just remove this line.

File: src/foam/core/NumberSet.js (hunk 2 above)
Comment (tharmaman) [reply src/foam/core/NumberSet.js]: Done.

File: src/foam/core/NumberSet.js (hunk 1 above)
Comment (tharmaman) [reply src/foam/core/NumberSet.js]: Done.

File: tools/classes.js (hunk 3 above)
Comment (tharmaman) [reply tools/classes.js]: Done.

**Pull request merged**